pkg/controller/podautoscaler/replica_calculator.go

```diff
@@ -107,9 +107,10 @@ func (c *ReplicaCalculator) GetResourceReplicas(ctx context.Context, currentRepl
 
 	if len(missingPods) > 0 {
 		if usageRatio < 1.0 {
-			// on a scale-down, treat missing pods as using 100% of the resource request
+			// on a scale-down, treat missing pods as using 100% (all) of the resource request
+			maxTargetUtilization := int64(max(100, int32(targetUtilization)))
```

Comment:
`int32(targetUtilization)` is redundant, `int64(max(100, targetUtilization))` should work.

Reply:
Addressed